packages/experimental/src/global/js/package-settings.js

```diff
@@ -47,17 +47,56 @@ const defaults = {
   },
 };
 
-const setAll = (flags, enabled) => {
-  for (const key in flags) {
-    flags[key] = enabled;
-  }
-};
+const component = { ...defaults.component };
+const feature = { ...defaults.feature };
+const warningMessage = (type, property) =>
+  `IBM Cloud Cognitive (WARNING): ${type} "${property}" enabled via feature flags. This component has not yet completed it's review process.`;
+const warningMessageAll = (type) =>
+  `IBM Cloud Cognitive (WARNING): All canary ${type.toLowerCase()} have been enabled through use of setAll${type}`;
```

Comment:
Ingenious, but I think it would be better to have separate messages for components and features. The message about components not having completed their review process is, I think, crucial, but doesn't apply to features so would seem a bit odd there.

```suggestion
const warningMessageComponent = property =>
  `IBM Cloud Cognitive (WARNING): Component "${property}" enabled via feature flags. This component has not yet completed its review process.`;
const warningMessageFeature = property =>
  `IBM Cloud Cognitive (WARNING): Feature "${property}" enabled via feature flags.`;
const warningMessageAllComponents =
  'IBM Cloud Cognitive (WARNING): All components enabled through use of setAllComponents. This includes components that have not yet completed their review process.';
const warningMessageAllFeatures = (type) =>
  'IBM Cloud Cognitive (WARNING): All features enabled through use of setAllFeatures';
```